EMIP/1-1000/EMIP00302.xml

```diff
@@ -37,9 +38,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </msIdentifier>
                <msContents>
                   <summary>Collection of Hymns of the Horologium</summary>
-                  <msItem xml:id="ms_i1">
                      <textLang mainLang="gez">Gə‘əz</textLang>
-                  </msItem>
+                     <msItem xml:id="ms_i1"><locus from="1v" to="70v"/><title type="complete" ref="LIT3575Mashaf"/>
```

Comment:
if the note says it is incomplete, way does the title element claims in type for completeness?